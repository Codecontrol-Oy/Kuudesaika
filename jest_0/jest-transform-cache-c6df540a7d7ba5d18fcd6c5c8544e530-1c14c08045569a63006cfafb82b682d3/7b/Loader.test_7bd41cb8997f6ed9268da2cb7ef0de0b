'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _Loader = require('./Loader');var _Loader2 = _interopRequireDefault(_Loader);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe('<Loader />', function () {
    it('should match snapshot', function () {
        var loader = shallow(_react2.default.createElement(_Loader2.default, null));
        expect(loader).toMatchSnapshot();
    });
    it('should contain correct elements', function () {
        var loader = shallow(_react2.default.createElement(_Loader2.default, null));
        expect(loader.find('span').length).toEqual(1);
        expect(loader.find('Spinning').length).toEqual(1);
    });
});