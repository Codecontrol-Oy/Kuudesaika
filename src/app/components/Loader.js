import React from 'react';
import Spinning from 'grommet/components/icons/Spinning';

export default class Loader extends React.Component {
  constructor (props) {
    super(props);
  }
  render () {
    return (
            <div>
              <Spinning size={"medium"} />
              <span>LADATAAN...</span>
            </div>
    );
  }
}
