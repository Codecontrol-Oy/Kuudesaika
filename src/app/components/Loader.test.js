import React from 'react';
import Loader from './Loader';

describe('<Loader />', () => {
    it('should match snapshot', () => {
        const loader = shallow(<Loader />);
        expect(loader).toMatchSnapshot();
    });
    it('should contain correct elements', () => {
        const loader = shallow(<Loader />);
        expect(loader.find('span').length).toEqual(1);
        expect(loader.find('Spinning').length).toEqual(1);
    });
});

