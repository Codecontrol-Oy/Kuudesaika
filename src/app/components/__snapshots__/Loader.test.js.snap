// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should match snapshot 1`] = `
<div>
  <Spinning
    responsive={true}
    size="medium"
  />
  <span>
    LADATAAN...
  </span>
</div>
`;
